src/node.rs

```diff
@@ -841,12 +975,19 @@ impl<S: ForkSource + std::fmt::Debug> InMemoryNode<S> {
             tx.payer()
         );
         println!(
-            "Gas - Limit: {:?} | Used: {:?} | Refunded: {:?}",
-            tx.gas_limit(),
-            tx.gas_limit() - tx_result.gas_refunded,
-            tx_result.gas_refunded
+            "Gas - Limit: {} | Used: {} | Refunded: {}",
+            to_human_size(tx.gas_limit()),
+            to_human_size(tx.gas_limit() - tx_result.gas_refunded),
+            to_human_size(tx_result.gas_refunded.into())
         );
 
+        if self
+            .display_detailed_gas_info(&vm, spent_on_pubdata)
```

Comment:
As there's a lot of information printed to the console about gas details, do we want to have a flag similar to `show-calls`?